pages/header.py

```diff
@@ -0,0 +1,20 @@
+from methods import Methods
+from locators import Locators
+
+class Header():
+    """This is parent class"""
+    def __init__(self, driver):
+        self.driver = driver
+        self.methods = Methods(self.driver)
+        self.locators = Locators
+
+    def click_icon(self):
+        """Clicks on the round icon"""
+        self.methods.click_element(self.locators.ICON)
+
+    def click_dropdown(self, pick_item):
```

Comment:
Looks like naming is wrong. click_dropdown -> select_option(self, option)

Reply:
good point